### DISCUSSION TITLE
A Time of War, please?

### DISCUSSION THREAD

**Discussion by SANSd20** [chummer5a/chummer5a#4979] [General]
Would the Devs be willing to look into making a character creator for _A Time of War: The Battletech RPG_ as there are similarities in the rule systems?

**Comment by chummer5a** [comment]
Not really. Would be happy to have someone else fork the app and make the
necessary changes, but as-is I don't see it happening by our team.

**Reply by SANSd20** [reply]
Thank you anyway. It was worth a shot to ask.
